### ISSUE TITLE
Error when adding new file with Subversion 1.8

### ISSUE THREAD

**Issue by kaazoo** [amartel/redmine_scm_extensions#16]
We recently migrated our Redmine system (version 2.4.6, SCM extensions plugin version 0.4) from Debian 6 (includes Subversion 1.6.12) to Ubuntu 14.04 (includes Suversion 1.8.8). Now, the upload file functionality gives a error message:
```
Error. Update canceled
```

Apache error log shows:
```
App 32138 stderr: svn: warning: W160013: URL 'file:///srv/subversion/project/testfile.txt' non-existent in revision 79
App 32138 stderr: svn: E200009: Could not list all targets because some targets don't exist
```

However, the file is still uploaded as a new revision. I think this is a compatibility issue with Subversion 1.8+.

**Comment by Mask** [comment]
does this error occur only on adding a new file, or also when updating an existing file?

**Comment by kaazoo** [comment]
This problem can be solved by updating Subversion to 1.8.13. There is a PPA für Ubuntu:
```
add-apt-repository ppa:dominik-stadler/subversion-1.8
aptitude update
aptitude upgrade
```
